CONTRIBUTING.md

```diff
@@ -21,7 +21,7 @@ TBD.
 ## Installing the development environment
 
 ```
-python -m pip install --ignore-installed -U -e .[complete]
+python -m pip install --upgrade --editable ".[all]"
```

Comment:
Henry had previously suggested to me in `pyhf` that `--ignore-installed` really isn't needed.